molecule/resources/verify.yml

```diff
@@ -5,17 +5,15 @@
   become: true
   gather_facts: false
 
+  vars:
+    openwisp2_monitoring: true
+
```

Comment:
I don't think we need this since the default value is `true`. Can you confirm @Purhan ?

Reply:
Yeah right, I will remove it :+1: